packages/graphql/tests/integration/aggregations/top_level/bigint.test.ts

```diff
@@ -0,0 +1,288 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Driver } from "neo4j-driver";
+import { graphql } from "graphql";
+import { generate } from "randomstring";
+import neo4j from "../../neo4j";
+import { Neo4jGraphQL } from "../../../../src/classes";
+
+describe("aggregations-top_level-bigint", () => {
+    let driver: Driver;
+
+    beforeAll(async () => {
+        driver = await neo4j();
+    });
+
+    afterAll(async () => {
+        await driver.close();
+    });
+
+    test("should return the min of node properties", async () => {
+        const session = driver.session();
+
+        const typeDefs = `
+            type Movie {
+                testString: String
+                imdbRatingBigInt: BigInt
+            }
+        `;
+
+        const testString = generate({
+            charset: "alphabetic",
+            readable: true,
+        });
+
+        const neoSchema = new Neo4jGraphQL({ typeDefs });
+
+        try {
+            await session.run(
+                `
+                    CREATE (:Movie {testString: $testString, imdbRatingBigInt: 1.1})
+                    CREATE (:Movie {testString: $testString, imdbRatingBigInt: 2.1})
+                    CREATE (:Movie {testString: $testString, imdbRatingBigInt: 3.1})
+                    CREATE (:Movie {testString: $testString, imdbRatingBigInt: 4.1})
```

Comment:
You've created this big integer values in all of these tests as floats - it's probably worth changing these to values over 2147483647 so that we can properly validate this behaviour.

Reply:
Yes!